content/en/docs/tasks/inject-data-application/podpreset.md

```diff
@@ -1,22 +1,19 @@
 ---
 reviewers:
 - jessfraz
-title: Inject Information into Pods Using a PodPreset
+title: Inject Information into Pods Using a Pod Preset
```

Comment:
I'd leave the original title

Reply:
I accepted this change in one of the last commits